lib/src/action.dart

```diff
@@ -45,15 +56,15 @@ import 'package:w_flux/src/constants.dart' show v3Deprecation;
 /// when a consumer needs to check state changes immediately after invoking an
 /// action.
 ///
-class Action<T> extends Object with Disposable implements Function {
+class ActionV2<T> extends Object with Disposable implements Function {
```

Comment:
#supernit. I think we could simplify this without affecting the public API.

```suggestion
class ActionV2<T> extends Disposable {
```